packages/eslint-plugin/src/utils/configHelpers.js

```diff
@@ -136,15 +136,27 @@ module.exports = {
        * typically include will not be specified that way
        *
        * @type {{
-          extends: string[];
+          extends: string;
           include: string[];
-          excludes: string[];
+          exclude: string[];
           compilerOptions: Record<string,unknown>;
           references?: Array<{path:string}>
           }}
        */
     const tsconfig = jju.parse(fs.readFileSync(tsconfigPath).toString());
 
+    /**
+     * Handle any necessary extends merging here, make sure to treat just like native tsconfigs would
```

Comment:
I wonder if we should merge any of the other extended stuff as well?

Reply:
I guess we could just do a spread. I think overwiriting the root properties is exactly with tsconfig normally does anyway

Reply:
smth like this
```ts
mergedTsConfig = {
  ...parentTsConfig,
  ...tsConfig
}
```

Reply:
Sort of, though you'd need separate merging for the compilerOptions. I'm not sure how it handles the other root properties.

Reply:
> It’s worth noting that files, include and exclude from the inheriting config file overwrite those from the base config file, and that circularity between configuration files is not allowed.

https://www.typescriptlang.org/tsconfig#extends

might as well follow what tsconfig extend does :D